src/cpdb/cpdb.py

```diff
@@ -11,13 +12,14 @@
 import plotly.express as px
 import plotly.graph_objects as go
 from src.constants import COLOR_SCHEME
+from src.cpdb.components.geometry_visualization_report import geometry_visualization_report
 from src.cpdb.components.adminbounds import adminbounds
 from src.cpdb.components.withinNYC_check import withinNYC_check
 
 
 def cpdb():
     st.title("Capital Projects Database QAQC")
-    branch = st.sidebar.selectbox("select a branch", ["main"])
+    branch = st.sidebar.selectbox("select a branch", ["main", "108-Convert-Lines-to-Polygons-Bug"])
```

Comment:
prob worth deleting before merging

Reply:
Yeah I am not able to view the page on the 108 branch